src/css/sass/colorschemes/cannabis.scss

```diff
@@ -7,27 +7,30 @@
 /* Colorscheme root variables */
 :root {
   /* Theme colors */
-  --primary-color: #064e66;
+  --primary-color: #064E66;
   --primary-dark-color: #043747;
   --primary-light-color: #518394;
-  --primary-lightest-color: #83a7b3;
+  --primary-lightest-color: #B4CAD1;
   --primary-hover-color: #043747;
-  // --primary-active-color: #4A1106;
-  // --primary-focused-color: #4A1106;
-  // --primary-disabled-color: #4A1106;
-  --secondary-color: #fec02f;
-  --secondary-dark-color: #222222;
-  --secondary-light-color: #6F6F6F;
-  --secondary-lightest-color: #D6D6D6;
-  --secondary-hover-color: #222222;
-  // --secondary-active-color: #4A1106;
-  // --secondary-focused-color: #4A1106;
-  // --secondary-disabled-color: #4A1106;
-  --highlight-color: #fdb81e;
-  --highlight-color-300: #FFF2D5;
 
-  --standout-color: #2f4c2c;
-  --standout-hover-color: #21351f; 
-  --logo-font: 'Lexend';
+  --secondary-color: #2F4C2C;
```

Comment:
are these color changes intentional or are they coming through from another update?

Reply:
Yep, I copied over the color changes from something else that Konstantin was doing for Cannabis, just to see correct colors in dev testing. I'll defer to Konstantin's changes when I handle the merge conflict, but looks like it's all the same.

Reply:
I did change the name of the --logo-font variable to match all the other variables. I didn't see any other uses of this in Cannabis, but we should probably check with @kkoryaka just to be sure.

Reply:
Resolved conflicts with main branch.